routing/edge_estimator.cpp

```diff
@@ -38,13 +38,12 @@ double CalcTrafficFactor(SpeedGroup speedGroup)
 {
   if (speedGroup == SpeedGroup::TempBlock)
   {
-    double constexpr kImpossibleDrivingFactor = 1e4;
-    return kImpossibleDrivingFactor;
+    // impossible driving factor
```

Comment:
С константой же было норм. Зачем убрал?
```suggestion
    // Impossible driving factor (поясни тогда уже что это)
```

Reply:
- Не понимаю для чего добавлять переменную, которая используется сразу же и только в следующей строке.
- Переменная стала комментарием, других пояснений нет.